src/gossip/fuzz_table.zig

```diff
@@ -4,17 +4,14 @@ const network = @import("zig-network");
 
 const AtomicBool = std.atomic.Value(bool);
 const KeyPair = std.crypto.sign.Ed25519.KeyPair;
-
```

Comment:
I'm always changing my mind about how to split things up but usually I like to keep a blank line between std and sig imports if there's more than a few imports.

Also usually I keep unbroken blocks of imports internally alphabetized. I can see arguments against doing that though if there are other logical separations. Or maybe it doesn't even matter and it's a waste of time.

Reply:
i dont have a strong preference - i agree its nice to keep the std with a blank line between - the imports below were spaced out for no reason i could tell so i removed them and the std line was a casualty

Reply:
2cb9de6